src/modules/coreHttpApi/openapi.yml

```diff
@@ -238,33 +238,11 @@ paths:
         Syncs the Connector messages and relationships with the Backbone.
         Checks for new relationships as well as incoming changes of existing ones.
         Checks for new or updated Messages.
-        Returns all affected relationships and messages.
       tags:
         - Account
       responses:
-        200:
-          description: Success
-          content:
-            application/json:
-              schema:
-                type: object
-                properties:
-                  result:
-                    type: object
-                    properties:
-                      relationships:
-                        type: array
-                        items:
-                          $ref: "#/components/schemas/Relationship"
-                      messages:
-                        type: array
-                        items:
-                          $ref: "#/components/schemas/Message"
-                    required:
-                      - messages
-                      - relationships
-                required:
-                  - result
+        204:
+          description: Success. Received new/changed Relationships/Messages, if present.
```

Comment:
```suggestion
          description: No content.
```

Reply:
I prefer the style of your recent 204 description of the attribute deletion which actually explains something.

Reply:
I don't.

Reply:
Especially if your description is only partially right.

Reply:
As is the description is very misleading.

Reply:
How?

Reply:
Sounds like you would receive new/changed Relationships/Messages, but you don't receive anything.